bindings/python/iota_sdk/types/context_input.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass, field
+from enum import IntEnum
+from iota_sdk.types.common import HexStr, json
+
+
+class ContextInputType(IntEnum):
+    """Context input types.
+    """
+    BlockIssuanceCredit = 1
+
+
+@json
+@dataclass
+class ContextInput():
+    """Base class for context inputs.
+    """
+    type: int
+
+
+@json
+@dataclass
+class BlockIssuanceCreditContextInput(ContextInput):
+    """A Block Issuance Credit (BIC) Context Input provides the VM with context for the value of
+    the BIC vector of a specific slot.
+    """
+    account_id: HexStr
```

Comment:
Will we have a proper type for this at some point? If yes, TODO

Reply:
Last time we said no